### PR TITLE
Fix #65: Add a thread view.

### LINKED ISSUE #65: Load full discussion when clicking on a message
When I click on a message, I want to see the full discussion in the right sidebar.

### PR DESCRIPTION
**Pull request by n1k0**
I'd like reviews from @vjousse and @nicosomb and/or @magopian.

### CODE REVIEW

**Comment by n1k0** [comment]
Note: I've also fixed a bunch of broken endpoints in https://github.com/n1k0/tooty/pull/73/commits/53e275c29a5a5cd393bf966ab899d5a430f8eaa4.

File: src/Model.elm [hunk 1]
```diff
@@ -99,11 +116,11 @@ type alias Model =
     , globalTimeline : List Mastodon.Model.Status
     , notifications : List Mastodon.Model.NotificationAggregate
     , draft : Draft
-    , account : Maybe Mastodon.Model.Account
     , errors : List String
     , location : Navigation.Location
     , useGlobalTimeline : Bool
     , viewer : Maybe Viewer
+    , currentView : CurrentView
```
Comment (vjousse) [inline src/Model.elm]: So, shouldn't we subscribed to WebSocket events based on this too?

Comment (n1k0) [inline src/Model.elm]: Yeah I was about to do that but I'd rather land that patch first then iterate.

Comment (n1k0) [inline src/Model.elm]: Filed #74 about that.

**Pull request merged**